tools/enso4igv/src/main/resources/org/enso/tools/enso4igv/enso.tmLanguage.json

```diff
@@ -21,7 +21,7 @@
 		"keywords": {
 			"patterns": [{
 					"name": "keyword.other",
-					"match": "\\b(here|this|type)\\b"
+					"match": "\\b(here|self|type)\\b"
```

Comment:
`here` can be removed - it is no longer a keyword.

Reply:
Right!